<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FinderWindow</class>
 <widget class="QDialog" name="FinderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>100</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Finder</string>
  </property>
  <layout class="QGridLayout" name="mFinderLayout">
   <item row="0" column="5">
    <widget class="QGroupBox" name="mTextGroupBox">
     <property name="title">
      <string>Text</string>
     </property>
     <layout class="QFormLayout" name="mTextGroupLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QLineEdit" name="mTextLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="mTextFrame1">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="mTextFrameLayout1">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="mTextPrevious">
           <property name="text">
            <string>Previous</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="mTextNext">
           <property name="text">
            <string>Next</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QFrame" name="mTextFrame2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="mTextFrameLayout2">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="mTextCaseSensitive">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>Case Sensitive</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QFrame" name="mButtonFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="mButtonLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="mButtonFind">
        <property name="text">
         <string>Find</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mButtonCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
